src/Apps/MyCollection/Routes/UploadFlow/ArtworkDetails/Mutations/CreateArtworkMutation.ts

```diff
@@ -0,0 +1,60 @@
+import { commitMutation, Environment, graphql } from "relay-runtime"
+import {
+  CreateArtworkMutation,
+  MyCollectionCreateArtworkInput,
+} from "__generated__/CreateArtworkMutation.graphql"
+
+export const createArtworkMutation = (
```

Comment:
Can we use the `useMutation` hook pattern as seen [here](https://github.com/artsy/force/blob/main/src/Apps/Settings/Routes/DeleteAccount/useDeleteAccount.ts)? (Please spread the word 🙏 )

Reply:
Looks way better 🌟